myuw-ios/ApplicationController.swift

```diff
@@ -101,10 +114,32 @@ class ApplicationController: UITabBarController, UITabBarControllerDelegate {
         }
         
         self.viewControllers = controllers
+                    
+        // set tabHome active
+        if lastTabIndex as! Int == 10 {
+            self.selectedViewController = moreNavigationController
+        } else {
+            self.selectedIndex = lastTabIndex as! Int
```

Comment:
The index needs to be an Int since that is one of the types that can be stored in UserDefaults. I tried storing the actual viewController itself... and that became very problematic. Simple answer... can't store viewControllers in UserDefaults vert easily.

I will need to clear the index (set to 0) whenever we get new affiliations.

I disabled tab reordering altogether... as it was getting to be a pain to persist that.